services/web-app/contexts/auth.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { createContext, useContext, useEffect, useState } from 'react'
+
+const AuthContext = createContext({})
+
+export const AuthProvider = ({ children }) => {
```

Comment:
Minor general style thing: Consider moving all exports to the bottom of the file in a single block.

Reply:
Done!